viite-UI/src/model/RoadAddressProjectCollection.js

```diff
@@ -142,7 +142,15 @@
         name: data[0].value,
         startDate: data[1].value,
         additionalInfo: data[2].value,
-        roadPartList: dirtyRoadPartList
+        roadPartList: _.map(dirtyRoadPartList, function(part){
```

Comment:
We should use only one list but have `dirty: (true | false)` for them and handle them in post that way